settings.py

```diff
@@ -1,73 +1,122 @@
-folder_dic = {
-    "Exomes": {
-        "input": "/hpc/diaggen/data/upload/Exomes/",
-        "output": "{}/Illumina/Exomes/".format(bgarray),
-        "files_required": ["workflow.done"],
-        "continue_without_email": False,
-        "upload_gatk_vcf": True,
-        "upload_exomedepth_vcf": True,
-    },
-    "Genomes": {
-        "input": "/hpc/diaggen/data/upload/Genomes/",
-        "output": "{}/Illumina/Genomes/".format(bgarray),
-        "files_required": ["workflow.done"],
-        "continue_without_email": False,
-        "upload_gatk_vcf": False,
-        "upload_exomedepth_vcf": False,
-    },
-    "TRANSFER": {
-        "input": "/hpc/diaggen/data/upload/TRANSFER/",
-        "output": "{}/TRANSFER/".format(bgarray),
-        "files_required": [""],
-        "continue_without_email": False,
-        "upload_gatk_vcf": False,
-        "upload_exomedepth_vcf": False,
-    },
-    "RAW_data": {
-        "input": "/hpc/diaggen/data/upload/RAW_data/",
-        "output": "{}/RAW_data/BACKUP_TEMP/".format(bgarray),
-        "files_required": [""],
-        "continue_without_email": False,
-        "upload_gatk_vcf": False,
-        "upload_exomedepth_vcf": False,
+transfer_settings = {
+    "bgarray": {
+        "mount_path": "/mnt/bgarray/",
+        "transfers": [
+            {
+                "name": "Exomes",
+                "input": "/hpc/diaggen/data/upload/Exomes/",
+                "output": "Illumina/Exomes/",
+                "files_required": ["workflow.done"],
+                "continue_without_email": False,
+                "upload_gatk_vcf": True,
+                "upload_exomedepth_vcf": True,
+            },
+            {
+                "name": "Genomes",
+                "input": "/hpc/diaggen/data/upload/Genomes/",
+                "output": "Illumina/Genomes/",
+                "files_required": ["workflow.done"],
+                "continue_without_email": False,
+                "upload_gatk_vcf": False,
+                "upload_exomedepth_vcf": False,
+            },
+            {
+                "name": "TRANSFER",
+                "input": "/hpc/diaggen/data/upload/TRANSFER/",
+                "output": "TRANSFER/",
+                "files_required": [""],
+                "continue_without_email": False,
+                "upload_gatk_vcf": False,
+                "upload_exomedepth_vcf": False,
+            },
+            {
+                "name": "RAW_data",
+                "input": "/hpc/diaggen/data/upload/RAW_data/",
+                "output": "RAW_data/BACKUP_TEMP/",
+                "files_required": [""],
+                "continue_without_email": False,
+                "upload_gatk_vcf": False,
+                "upload_exomedepth_vcf": False,
+            },
+            {
+                "name": "Transcriptomes",
+                "input": "/hpc/diaggen/data/upload/Transcriptomes/",
+                "output": "Illumina/Research_Dx/Transcriptomes/UMCU_hg38/",
```

Comment:
Dit lijkt mij niet de juiste folder voor in prodcutie wat het in ten slotte geen research meer na de implementatie toch?
Dan zou ik keizen voor /mnt/bgarray/Illumina/Transcriptomes/ of /mnt/bgarray/Illumina/RNAseq/

Reply:
Fixed in [cb74422](https://github.com/UMCUGenetics/rsync_to_rdisc/pull/44/commits/cb74422df324e81080b411c12a4b28cf3d17e778) en [f9697f5](https://github.com/UMCUGenetics/rsync_to_rdisc/pull/44/commits/f9697f542c61d0277aa2015f719537536d6840fd)